### PR TITLE
Add prebuilt bindings for GDAL 3.6 and document how to do it

### LINKED ISSUE #351: Add pre-built bindings for GDAL 3.6
```
  thread 'main' panicked at 'No pre-built bindings available for GDAL version 3.6. Use `--features bindgen` to generate your own bindings.', /home/thomas/.cargo/registry/src/github.com-1ecc6299db9ec823/gdal-sys-0.7.0/build.rs:248:17
```

Yep, these [aren't currently included](https://github.com/georust/gdal/tree/548cd33f392dd65a5858368baa9c3b70b40956f1/gdal-sys/prebuilt-bindings).

It works if I put this in my `Cargo.toml`:

```
gdal-sys = { version = '0.8.0', features = ['bindgen'] }
```

I tried to create a PR for prebuilt bindings for 3.6 by copying the `bindings.rs` file from somewhere inside the `target/` directory, but it seems to be missing docstrings that `gdal_3.5.rs` has. If someone would tell me (or better yet, document!) how to generate these properly, I'll be happy to finish the PR.

### PR DESCRIPTION
**Pull request by ttencate**
Fixes #351

- [x] I agree to follow the project's [code of conduct](https://github.com/georust/gdal/blob/master/CODE_OF_CONDUCT.md).
- [x] I added an entry to `CHANGES.md` if knowledge of this change could be valuable to users.
---

### CODE REVIEW

**Comment by ttencate** [comment]
Looks like GitHub didn't run my newly added CI check. I don't know how to trigger that.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -12,6 +12,46 @@ env:
   RUST_BACKTRACE: 1
 
 jobs:
+  gdal_36:
+    name: "ci gdal-34"
```
[the remark this answers is not in the document]
Comment (ttencate) [reply .github/workflows/ci.yml]: Just checking if anyone was paying attention ;)

**Comment by lnicola** [comment]
Oops, sorry, I've already filed #353.

**Comment by ttencate** [comment]
> Oops, sorry, I've already filed #353. There's a bit more to change that the `fs::write` call.

I'll remove my last commit and leave that to you to fix then.

**Review by lnicola — APPROVED** [review]

**Comment by jcphill** [comment]
FYI top-level README.md still says 3.5.

**Pull request merged**